readme.md

```diff
@@ -230,6 +268,8 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+db.instructors.find(name: "Andy")
+db.instructors.wishlist_items.insert({
+  description: "Gold Deer"
+});
 ```
-
```

Comment:
True, we would use that `db` when running Mongoose commands from the command line. But how would you do this assuming you have access to an `Instructor` class?